scripts/evalCrawler.R

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env Rscript
+
+library(stringr)
+library(readr)
+library(purrr)
+library(dplyr)
+library(pbapply)
+
+#Sys.setlocale('LC_ALL','C')
+regex <- list()
```

Comment:
```r
regex <- list(
  c("(?<!([\"a-zA-Z0-9_#]))(eval\\()" , "4" , "eval"),
  c("(?<!([\"a-zA-Z0-9_#]))(evalq\\()" , "5" , "evalq"),
  ...
)
```

consider to put the prefix in some constant.

Reply:
But anyway, why do you need a list?

Reply:
Acknowledged!
List - to apply the same function to each member of the list using `lapply`